OsmAnd/res/values/strings.xml

```diff
@@ -142,27 +142,27 @@
+    <string name="last_launch_crashed">Last OsmAnd run crashed. Please help us improve OsmAnd by sharing the error message.</string>
+    <string name="press_again_to_change_the_map_orientation">Tap again to change map orientation</string>
     <string name="process_downloading_service">OsmAnd downloading service</string>
     <string name="shared_string_color_magenta">Magenta</string>
     <string name="shared_string_icon">Icon</string>
     <string name="rate_dialog_descr">Please give us 30 seconds by sharing your feedback and rating our work on Google Play.</string>
     <string name="button_rate">Rate</string>
     <string name="shared_string_privacy_policy">Privacy Policy</string>
-    <string name="help_us_make_osmand_better">Help us make OsmAnd better!</string>
-    <string name="make_osmand_better_descr">Allow OsmAnd to collect and process anonymous application usage data. We do not collect or store data about your position, or about any locations you view on the map.\n\nYou can change your selection at any time in Settings > Privacy and Security.</string>
-    <string name="choose_data_to_share">Choose what type of data you want to share:</string>
+    <string name="help_us_make_osmand_better">Help us make OsmAnd better</string>
+    <string name="make_osmand_better_descr">Allow OsmAnd to collect and process anonymous app usage data. No data about your position, or locations you view on the map are collected.\n\nIt is always available in \'Settings\' → \'Privacy and Security\'.</string>
+    <string name="choose_data_to_share">Choose the type of data you want to share:</string>
     <string name="downloaded_maps">Maps downloaded</string>
     <string name="visited_screens">Screens visited</string>
     <string name="collected_data">Data collected</string>
     <string name="collected_data_descr">Define which data you allow OsmAnd to share.</string>
-    <string name="downloaded_maps_collect_descr">Helps us understand which maps of which regions and countries are the most popular.</string>
-    <string name="visited_screens_collect_descr">Helps us understand which OsmAnd features are the most popular.</string>
-    <string name="privacy_and_security_change_descr">By tapping \"Allow\" you agree with our %1$s</string>
+    <string name="downloaded_maps_collect_descr">Helps us understand country and region map popularity.</string>
+    <string name="visited_screens_collect_descr">Helps us understand OsmAnd feature popularity.</string>
+    <string name="privacy_and_security_change_descr">By tapping \"Allow\", you agree with our %1$s</string>
     <string name="settings_privacy_and_security">Privacy and security</string>
-    <string name="settings_privacy_and_security_desc">Select which data you are sharing with us</string>
-    <string name="shared_string_no_thank_you">No, thank you</string>
+    <string name="settings_privacy_and_security_desc">Select what data you share</string>
+    <string name="shared_string_no_thank_you">No</string>
```

Comment:
In an instructing to the non-sentient, I find it misplaced. The "No" is the standard, and it will probably end up translated where it does not belong at all.

Reply:
Let us please leave it unchanged for now, until we positively identify/evaluate  all places where the string is used